security/qatlib_rbac.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: intel-qat
+  namespace: intel-qat
+---
+apiVersion: security.openshift.io/v1
```

Comment:
Could we use a separate file called qatlib_scc.yaml for intel-qat-scc? and split the PR into two.
One is for intel-qat-scc another is for qatlib_rbac.yaml used to show users how to use the user defined scc.
In the PR for intel-qat-scc commit log. We should clarify that this Intel-qat-scc is built from OCP predefined SCC restrict-v2 with the IPC_Lock capability and RunAsAny permission added.

Reply:
if we keep them separate, there will be multiple yaml's for the user to deploy for qatlib. or we remove rbac.yaml and add `oc adm `command to bind scc to a service account

Reply:
Yes, we need one more command to create the intel-qat-scc. But the SCC definition is easy to maintain in a separate file. And it is also easy for users to use it to create different service accounts. Let's do that. Thanks!

Reply:
updated.